docs/docs/index.md

```diff
@@ -1,36 +1,24 @@
-# Welcome
+# **Welcome**
```

Comment:
Bold on headlines is not having any effect

```suggestion
# Welcome
```

Reply:
reverted